os/netstat/netstat_linux.go

```diff
@@ -31,9 +41,16 @@ type TCPStat struct {
 	RetransSegs  *metrics.Counter
 	InErrs       *metrics.Counter
 	OutRsts      *metrics.Counter
-	Extended     *ExtendedMetrics
-	// not exported
-	m *metrics.MetricContext
+}
+
+// UDPStat represents statistics about UDP. Reflection is used to map field names to
+// kernel data.
+type UDPStat struct {
+	InDatagrams  *metrics.Counter
+	InErrors     *metrics.Counter
+	OutDatagrams *metrics.Counter
+	RcvbufErrors *metrics.Counter
+	SndbufErrors *metrics.Counter
```

Comment:
Based on reading this blog: https://blog.packagecloud.io/eng/2016/06/22/monitoring-tuning-linux-networking-stack-receiving-data/#procnetsnmp